### PR TITLE
reset accumulated output text

### PR DESCRIPTION
**Pull request by hugobho**
Reset accumulated output text
Feature tests written

### CODE REVIEW

**Comment by hugobho** [comment]
@kevgo

File: features/reset-stream.feature [hunk 1]
```diff
@@ -0,0 +1,14 @@
+Feature: Resetting accumulated text stream
+
+  As a developer dealing with an output stream
+  I want to be able to clear all previously accumulated text
+  So that I may search through only new and relevant data.
+
+  Rules:
+  - call ".resetStream()" of the observable process to clear all accumulated output text
```
Comment (kevgo) [inline features/reset-stream.feature]: Let's call this `.resetOutputStream`

**Comment by kevgo** [comment]
Nice!

File: features/reset-stream.feature (hunk 1 above)
Comment (hugobho) [inline features/reset-stream.feature]: Should this be singular since there's only one output stream?

**Comment by hugobho** [comment]
@kevgo

File: features/reset-stream.feature [hunk 2]
```diff
@@ -0,0 +1,14 @@
+Feature: Resetting accumulated text stream
+
+  As a developer dealing with an output stream
+  I want to be able to clear all previously accumulated text
+  So that I may search through only new and relevant data.
+
+  Rules:
+  - call ".resetOutputStreams()" of the observable process to clear all accumulated output text
```
Comment (kevgo) [inline features/reset-stream.feature]: Remove "of the observable process", I think that's implicit enough here.

**Comment by kevgo** [comment]
🚢

**Pull request closed without merging**